spec/pipelines/ont/message_spec.rb

```diff
@@ -7,24 +7,23 @@
     create(:ont_min_know_version, name: 'v22', default: true)
   end
 
-  let(:config)            { Pipelines.configure(Pipelines.load_yaml) }
-  let(:pipeline_config)   { config.ont.message }
+  let(:configuration) { Pipelines.configure(Pipelines.load_yaml).ont.message }
```

Comment:
I would remove `.ont.message` from here or rename the variable to ont_message_config just to make it slightly clearer as 'configuration' is the high level class for the whole config.